### DISCUSSION TITLE
Ice Preference Menu doesn't appear when clicking on "ice settings"

### DISCUSSION THREAD

**Discussion by yoBoss283** [jordanbaird/Ice#399] [Q&A] [answered]
action: go to top menu, two tap click on ice divider to show menu, tap on "ice settings" > nothing happens.

I am not sure why it works sometimes. But I have experienced this since I have installed another user on my MacBook. Previously it was only 1 user. The other user can't open the "ice settings", whereas on my main device it depends when I restarted the machine. Is there perhaps something I am doing wrong here? or do I need to reinstall it in some way?

**Comment by jordanbaird** [comment]
What version if Ice are you on? Right click the app icon in Finder and go to "Get info". A version number should be in the panel that appears.

**Comment by yoBoss283** [comment]
It says 0.11.9Today it works fine? Not sure if it was auto updated? This issue was happening in the last couple of weeks or more.On Oct 10, 2024, at 16:37, Jordan Baird ***@***.***> wrote:﻿
What version if Ice are you on? Right click the app icon in Finder and go to "Get info". A version number should be in the panel that appears.

**Comment by jordanbaird** [comment, marked as answer]
This bug was fixed in 0.11.7. If you find that it somehow continues to be an issue, please file a bug report in the issues section of the repository.
